### ISSUE TITLE
Second Tab Always Selected

### ISSUE THREAD

**Issue by Vongalin** [igorlogius/bulk-open-focus-last#1]
Hi! Thank you for making this addon. Prior to it showing up on AMO it was working perfectly (loading as a temporary addon). Not sure when it stopped working, but after adding it from the official addon site and updating to the latest, it always selects the second tab that is created, no matter how many tabs are in the folder/group.

Example, I have a group with 8 YT links in it. Right-click, Open all Bookmarks, the second tab is always selected. I have another one with other links in it, only 3 and the second tab is selected.

I think it must be something related to how the other tabs are still loading. From what I can "see", the first tab loads, the second tab loads and gets selected because the others are still loading.

**Comment by igorlogius** [comment]
Yeah, i did fix an issue that would always cause singular ceated tabs (via CTRL-Click aka. opened in the background) to get the focus.  But the fix  is based on a timing bases solution which might not be as stable as i hoped it was. 

That said try to reproduce your issue and see if i can find a solution

**Comment by igorlogius** [comment]
I was able to recreate the issue (but only with my own heavily modified profile) which allowed me to create a workaround ... but kind of did not allow me to find the exact cause of the issue.  If i had to guess the onCreated and update event might be a unreliable combination (for a readon i dont know)  For now i am using the tabs highlight function ... which seems to work as expected in both cases.

**Comment by igorlogius** [comment]
fixed in version >= 1.1.6  
AMO deplyoment done

**Comment by Vongalin** [comment]
I checked the latest release on a new profile and it works perfectly. Next, in my full profile, I tested it with disabling each addon, one at a time (don't have many) and was able to find the culprit - "Temporary Containers". With this addon enabled it no longer focuses the last tab. I checked the options in "Temporary Containers" and do not see anything related to focusing or moving tabs. The only custom option I have enabled in it is "Automatic Mode" which must be somehow interfering with the normal tab opening and focusing.
